### ISSUE TITLE
varnish cache should pay attention to 301 vs 302 from archive

### ISSUE THREAD

**Issue by reedstrm** [Connexions/cnx-deploy#508]
## Summary

Currently, varnish treats anything > 300 from a backend as "do not cache" This is wrong for 301, which is "permanently moved" We are releasing changes to archive that make more URLs 301s. Our cache should distinguish, and cache those for a long time, rather than go to the backend every time.
## Environment
 This applies to all enviornments that deploy varnish

**Comment by BryanEliDimas** [comment]
@brittweinstein FYI, this is now in progress.

**Comment by BryanEliDimas** [comment]
Oh, actually @brittweinstein, I think this is the bigger issue that I'm supposed to be working on: https://github.com/openstax/cnx/issues/209

I'm still learning about the how these two relate.

**Comment by BryanEliDimas** [comment]
Yep, confirmed, this is a different issue 😅 . I'm working on cnx#209. Sorry for the confusion. @brittweinstein
